gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/component/GbAjaxLink.java

```diff
@@ -0,0 +1,29 @@
+package org.sakaiproject.gradebookng.tool.component;
+
+import org.apache.wicket.ajax.AjaxChannel;
+import org.apache.wicket.ajax.attributes.AjaxCallListener;
+import org.apache.wicket.ajax.attributes.AjaxRequestAttributes;
+import org.apache.wicket.ajax.markup.html.AjaxLink;
+import org.apache.wicket.model.IModel;
+
+/**
+ * A GradebookNG implementation of Wicket's AjaxLink that
+ * disables the link on click to reduce the likelihood of
+ * double-clicks resulting in double actions.
+ */
+abstract public class GbAjaxLink<T> extends AjaxLink {
+
+	public GbAjaxLink(String id) {
+		this(id, (IModel)null);
```

Comment:
Hm yeah.. I copied from the AjaxLink class, but you're totally right, the cast/null is not required.